tests/lib/rules/sort-vars.js

```diff
@@ -113,34 +113,109 @@ ruleTester.run("sort-vars", rule, {
         }
     ],
     invalid: [
-        { code: "var b, a", errors: [expectedError] },
-        { code: "var b=10, a=20;", errors: [expectedError] },
-        { code: "var all=10, a = 1", errors: [expectedError] },
-        { code: "var b, c, a, d", errors: [expectedError] },
-        { code: "var c, d, a, b", errors: 2 },
-        { code: "var a, A;", errors: [expectedError] },
-        { code: "var a, B;", errors: [expectedError] },
-        { code: "var a, B, c;", errors: [expectedError] },
-        { code: "var B, a;", options: ignoreCaseArgs, errors: [expectedError] },
-        { code: "var B, A, c;", options: ignoreCaseArgs, errors: [expectedError] },
+        {
+            code: "var b, a",
+            output: "var a, b",
+            errors: [expectedError]
+        },
+        {
+            code: "var b , a",
+            output: "var a , b",
+            errors: [expectedError]
+        },
+        {
+            code: [
+                "var b,",
+                "    a;"
+            ].join("\n"),
+            output: [
+                "var a,",
+                "    b;"
+            ].join("\n"),
+            errors: [expectedError]
+        },
+        {
+            code: "var b=10, a=20;",
+            output: "var a=20, b=10;",
+            errors: [expectedError]
+        },
+        {
+            code: "var b=10, a=20, c=30;",
+            output: "var a=20, b=10, c=30;",
+            errors: [expectedError]
+        },
+        {
+            code: "var all=10, a = 1",
+            output: "var a = 1, all=10",
+            errors: [expectedError]
+        },
+        {
+            code: "var b, c, a, d",
+            output: "var a, b, c, d",
+            errors: [expectedError]
+        },
+        {
+            code: "var c, d, a, b",
+            output: "var a, b, c, d",
+            errors: 2
```

Comment:
is this intentional?

Reply:
Yes, this test case was already included; I only expanded it and added the expected fixed `output`.

It is expected to have two errors per the documentation:

> Alphabetical list is maintained starting from the first variable and excluding any that are considered problems. So the following code will produce two problems:
> `var c, d, a, b;`
> But this one, will only produce one:
> `var c, d, a, e;`

Reply:
Do you mean this: `errors: [expectedError, expectedError]`

Reply:
I'm a little confused. Are you suggesting that the rule's functionality be changed to only report one error?

Reply:
I guess reporting one error can simplify the code: 
`check is sorted => check is safe to fix => do fix`. 
maybe just my personal preference, I don't feel strong.